Tools/CISupport/kill-old-processes

```diff
@@ -217,6 +216,9 @@ def main(user=None):
             os.system("killall -9 -v " + task)
         os.system("ps aux | grep -P '.+/python3? .+(run_webkit_tests|run-webkit-tests|mod_pywebsocket)' | grep -v grep | awk '{print $2}' | xargs -r kill")
         removeOrphanShmSegments()
+
+        # Some bots use a Xvfb instance on :0 for dbus activation, so don't kill it.
+        os.system("ps aux | grep -P 'Xvfb' | grep -v ' :0' | grep -v grep | awk '{print $2}' | xargs -r kill")
```

Comment:
This could still end killing the xvfb process that starts the bot main script. It is not guaranteed that the assigned screen to this first xvfb process is zero. It can have another number, specially if the service is restarted. We run the process with ```xvfb-run -a``` (the `-a` parameter means to assign to it automatically the first free screen that may be `0` or another number).

In order to differentiate from the `Xvfb` process launched by the test suite (the ones we want to kill) vs the ones launched by the bot startup helpers I suggest to grep also for the ```displayfd``` parameter that we pass on the test suite but is not passed on a normal usage of `Xvfb` with `xvfb-run`.

Suggested line:
`
 os.system("ps aux | grep -P 'Xvfb -displayfd [0-9]'  | awk '{print $2}' | xargs -r kill")`

Just a tip: when you want to filter grep itself from the output of grepping the process is enough with adding a regular expression to the expression grepped.
```ps aux | grep -P bash``` will show the grep process itself but ```ps aux | grep -P bas[h]``` will not. That works because with the regular expression the name of the process ```grep -P bas[h]``` not longer matches the regular expression (`bas[h]` as a literal string doesn't match `bas[h]` as a regular expression, but `bash` does)